crypto/ec/curve448/arch_32/f_impl32.c

```diff
@@ -100,6 +100,6 @@ void gf_mulw_unsigned(gf_s * RESTRICT cs, const gf as, uint32_t b)
 
 void gf_sqr(gf_s * RESTRICT cs, const gf as)
```

Comment:
Could you please rename also this one?

Reply:
I will follow the same standard, `ossl_gf_sqr`